lib/inspec/base_cli.rb

```diff
@@ -85,6 +85,8 @@ def self.exec_options
         desc: 'Show progress while executing tests.'
       option :distinct_exit, type: :boolean, default: true,
         desc: 'Exit with code 101 if any tests fail, and 100 if any are skipped (default).  If disabled, exit 0 on skips and 1 for failures.'
+      option :insecure, type: :boolean, default: false,
```

Comment:
Do you think this would be better in the "target_options" above instead? I think that will lay out better in the help menu as this only pertains to the target.